### PR TITLE
Defined RandomEngine(uint8_t) class

### PR DESCRIPTION
**Pull request by RedocamaI**

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -75,5 +78,50 @@ int main(int argc, char *argv[]){
         }
     }
     
+
+    // Generate the seed values:
```
Comment (RedocamaI) [inline src/main.cpp:82]: Worked for RandomEngine(uint8_t). We can discuss like this.

File: src/engine.h [hunk 2]
```diff
@@ -1,4 +1,26 @@
 // Define this class here.
+#include <vector>
+#include <string>
 class RandomEngine {
+    
+    public:
+        uint8_t length = 0;
+        bool symbl = false;
+        
+        std::vector<uint8_t>indx = {};
```
Comment (kumarharsh2396) [inline src/engine.h:10]: How vector array is used here.

Comment (kumarharsh2396) [reply src/engine.h:10]: Means need of this
        std::vector<uint8_t>indx = {};
```
```

Comment (div-sahu55) [reply src/engine.h:10]: the data type for this vector is uint8_t and its name is indx. thats all.

**Pull request merged**